src/shims/unix/fd.rs

```diff
@@ -48,6 +51,7 @@ pub trait FileDescription: std::fmt::Debug + Any {
 
     fn close<'tcx>(
         self: Box<Self>,
+        _ecx: &mut MiriInterpCx<'tcx>,
```

Comment:
`read` and `write` take `ecx` as last argument -- please make it consistently the second or the last argument everywhere.